src/CommunityToolkit.Maui.Core/Views/RoundedStackView.macios.cs

```diff
@@ -6,7 +6,7 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// <summary>
 /// A rounded <see cref="UIStackView"/>
 /// </summary>
-[Obsolete]
+[Obsolete("Use RoundedView")]
```

Comment:
Can we add the [`EditorBrowsableAttribute`](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.editorbrowsableattribute?view=net-7.0) as well?